src/main/java/uk/nhs/ctp/entities/Cases.java

```diff
@@ -20,7 +20,7 @@
 @Entity
 @Table(name = "cases")
 @Data
-public class Cases {
+public class Cases extends SupplierPartitioned {
```

Comment:
If we want to search for encounters by supplier, it might be useful to add an index on supplier ID.
Most tables won't need this unless there is a search method implemented.

Reply:
The next sub-task for this story is to update queries and indexes so can be done as part of that